### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by jzawadzki**
Hi!

I'm sending full Polish translation of symfony demo.
I will try to keep up with any changes needed in the future.

JZ

### CODE REVIEW

File: app/Resources/translations/messages.pl.xliff [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+                <source>note</source>
+                <target>UWAGA</target>
+            </trans-unit>
+            <trans-unit id="tip">
+                <source>tip</source>
+                <target>PORADA</target>
+            </trans-unit>
+            <trans-unit id="not_available">
+                <source>not_available</source>
+                <target>nie dostępne</target>
```
Comment (javiereguiluz) [inline app/Resources/translations/messages.pl.xliff]: This should probably be `Nie dostępne`

Comment (jzawadzki) [inline app/Resources/translations/messages.pl.xliff]: haha - sorry - correct is "niedostępne" - fixed

File: app/Resources/translations/validators.pl.xliff [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
```
Comment (javiereguiluz) [inline app/Resources/translations/validators.pl.xliff]: `target-language="en"` -> `target-language="pl"`

Comment (jzawadzki) [inline app/Resources/translations/validators.pl.xliff]: fixed

File: app/Resources/translations/messages.pl.xliff [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+                <source>title.source_code</source>
+                <target>Kod źródłowy użyty do wygenerowania tej podstrony</target>
+            </trans-unit>
+            <trans-unit id="title.controller_code">
+                <source>title.controller_code</source>
+                <target>Kod kontrolera</target>
+            </trans-unit>
+            <trans-unit id="title.twig_template_code">
+                <source>title.twig_template_code</source>
+                <target>Kod szablonu Twig</target>
```
Comment (javiereguiluz) [inline app/Resources/translations/messages.pl.xliff]: I can't speak Polish, but I just want to ask you if `szablonu Twig` is the correct term in this case. In the Polish docs is more common to find `szablon Twig` (singular) or `szablony Twig` (plural). Thanks!

Comment (jzawadzki) [inline app/Resources/translations/messages.pl.xliff]: @javiereguiluz "szablonu" is genitive form of singular word "szablon" (template)
In exact translation:
"template" -> "szablon"
"template code" -> "kod szablonu"

Comment (javiereguiluz) [inline app/Resources/translations/messages.pl.xliff]: Great! Thanks for the explanation.

**Comment by javiereguiluz** [comment]
Thanks @jzawadzki.

**Pull request closed without merging**